op-node/p2p/gossip.go

```diff
@@ -69,10 +70,14 @@ func blocksTopicV1(cfg *rollup.Config) string {
 	return fmt.Sprintf("/kroma/%s/0/blocks", cfg.L2ChainID.String())
 }
 
+func blocksTopicV2(cfg *rollup.Config) string {
+	return fmt.Sprintf("/optimism/%s/1/blocks", cfg.L2ChainID.String())
```

Comment:
Wouldn't it be nice to unify the format for v1, v2 topics?

Reply:
I agree